### ISSUE TITLE
Make Response status code accordions clickable

### ISSUE THREAD

**Issue by lornajane** [Redocly/redoc#1280]
As part of #1279 , we should revisit the expandable response sections. Firefox's accessibility tool suggests:

> Clickable elements must be focusable and should have interactive semantics. [Learn more](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#Clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

As a keyboard-only user, I can't focus this element, so I can't interact with it. It looks like a `<div>` tag with some `onClick` attached to it. A focusable element such as `<a>` or `<button>` would help here.

![Screenshot from 2020-05-29 10-35-26](https://user-images.githubusercontent.com/172607/83245355-85327980-a198-11ea-83db-a8b3fe2ada93.png)

For bonus points while you are looking at this:
 * The green text on a green background is causing "Does not meet WCAG standards for accessible text", I assume because the contrast is quite minimal.
 * The "graphic" (I think this is the polygon expanding arrow thing) should have a text label "Content with images must be labelled".
